packages/agent/conf/examples/oid4vci_metadata/knb_babylon.json

```diff
@@ -0,0 +1,168 @@
+              "alt_text": "Babylon bank logo"
+            },
+            "background_image": {
+              "url": "https://i.ibb.co/CHqjxrJ/Babylon_bank_bg.png",
+              "alt_text": "Waves in shades of blue"
+            }
+          },
+          {
+            "locale": "en-US",
+            "name": "Bank account credential",
+            "description": "Bank account credential",
+            "background_color": "rgba(0, 0, 0, 0.2)",
+            "text_color": "#FBFBFB",
+            "logo": {
+              "url": "https://dutchblockchaincoalition.org/assets/images/icons/Babylon_bank_logo.png",
+              "alt_text": "Babylon bank logo"
+            },
+            "background_image": {
+              "url": "https://i.ibb.co/CHqjxrJ/Babylon_bank_bg.png",
+              "alt_text": "Waves in shades of blue"
+            }
+          },
+          {
+            "locale": "nl-NL",
+            "name": "Bankrekeninggegevens",
+            "description": "Bankrekeninggegevens",
+            "background_color": "rgba(0, 0, 0, 0.2)",
+            "text_color": "#FBFBFB",
+            "logo": {
+              "url": "https://dutchblockchaincoalition.org/assets/images/icons/Babylon_bank_logo.png",
+              "alt_text": "Babylon bank logo"
+            },
+            "background_image": {
+              "url": "https://i.ibb.co/CHqjxrJ/Babylon_bank_bg.png",
+              "alt_text": "Waves in shades of blue"
+            }
+          }
+        ],
+        "order": [
+          "description",
+          "issuerName",
+          "companyName",
+          "bankAccountNumber",
+          "kvkNumber"
+        ],
+        "credentialSubject": {
+          "description": {
+            "value_type": "string",
+            "display": [
+              {
+                "name": "Description"
+              },
+              {
+                "name": "Description",
+                "locale": "en-US"
+              },
+              {
+                "name": "Beschrijving",
+                "locale": "nl-NL"
+              }
+            ]
+          },
+          "issuerName": {
+            "value_type": "string",
+            "display": [
+              {
+                "name": "IssuerName"
+              },
+              {
+                "name": "IssuerName",
+                "locale": "en-US"
+              },
+              {
+                "name": "NaamVanDeUitgever",
+                "locale": "nl-NL"
+              }
+            ]
+          },
+          "companyName": {
+            "value_type": "string",
+            "display": [
+              {
+                "name": "CompanyName"
+              },
+              {
+                "name": "CompanyName",
+                "locale": "en-US"
+              },
+              {
+                "name": "Bedrijfsnaam",
+                "locale": "nl-NL"
+              }
+            ]
+          },
+          "kvkNumber": {
```

Comment:
The domain specific keys need to be in dutch, given we will create the context in dutch. So with the exception of the issuer this should be:
kvknummer, bedrijdsnaam, bankrekeningnummer

Reply:
Can I assume that you mean description instead of issuer? Fixed

Reply:
No the issuer object in there